docs/sources/developers/plugins/build-a-streaming-data-source-plugin.md

```diff
@@ -6,27 +6,35 @@ title: Build a streaming data source plugin
 
 This guide explains how to build a streaming data source plugin.
```

Comment:
Do you need this sentence? Your paragraph below this one provides a nice intro.

Reply:
Done, will check fix in soon

Reply:
Checked in fix, available for review.